mandatory/4-tax.js

```diff
@@ -17,7 +19,11 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(tax) {
+  let newPrice = calculateSalesTax(tax);
+  let showPrice = newPrice.toFixed(2);
+  return "£" + showPrice;
+}
 
```

Comment:
That's nice, I just think maybe better to rename your input parameter. Maybe price rather than tax.

Reply:
Thanks Leila!